edm.yaml

```diff
@@ -192,21 +192,41 @@ datatypes :
      - fcc::CaloHit hits // Hits used in the cluster
 
   fcc::TrackState:
-    Description: "Track state at a given point on the track."
+    Description: "Track state at a given point on the track. The track parameters are chosen 
```

Comment:
Hello Colin,
To make it more consistent I could instead use:
float phi
float theta
float q/p
float [15] cov

and the two local coordinates are zero per definiton.